lib/parse.js

```diff
@@ -919,13 +925,21 @@ function parse($TEXT, options) {
         if (!is("punc", ";")) {
             init = is("keyword", "var")
                 ? (next(), var_(true))
-                : expression(true, true);
+                : is("keyword", "let")
+                ? (next(), let_(true))
```

Comment:
As of ES6, these declarations are not longer allowed to have initialisers. Also, they must be limited to a single declarator.